spec/system/admin/reports_spec.rb

```diff
@@ -295,8 +295,8 @@
       }
 
       before do
-        Timecop.travel(Time.zone.local(2013, 4, 25, 14, 0, 0)) { order1.finalize! }
-        Timecop.travel(Time.zone.local(2013, 4, 25, 16, 0, 0)) { order2.finalize! }
+        Timecop.travel(Time.zone.local(2021, 4, 25, 14, 0, 0)) { order1.finalize! }
+        Timecop.travel(Time.zone.local(2021, 4, 25, 16, 0, 0)) { order2.finalize! }
```

Comment:
So we'll have the same problem in a few years time?

I'm wondering if we can use the current date and just travel to a different time.

Reply:
> So we'll have the same problem in a few years time?
Yes, good point.

> I'm wondering if we can use the current date and just travel to a different time.
That's a good idea. I'll try to figure out the best way to do that.